tests/assets/cli/multi_app_norich.py

```diff
@@ -0,0 +1,37 @@
+import typer
+
+sub_app = typer.Typer()
+
+variable = "Some text"
+
+
+@sub_app.command()
+def hello(name: str = "World", age: int = typer.Option(0, help="The age of the user")):
+    """
+    Say Hello
+    """
+
+
+@sub_app.command()
+def hi(user: str = typer.Argument("World", help="The name of the user to greet")):
+    """
+    Say Hi
+    """
+
+
+@sub_app.command()
+def bye():
+    """
+    Say bye
+    """
+
+
+app = typer.Typer(help="Demo App", epilog="The end", rich_markup_mode=None)
```

Comment:
The point of this new test is this: checking that `rich_markup_mode=None`, with `rich` installed, still produces correct results (showing [required], default values, etc)